templates/full-time-program.html

```diff
@@ -189,9 +189,10 @@ <h1>Eligibility</h1>
         <li>are willing to be part of the program during Pacific Time hours</li>
         <li>are dedicated to building a long-term career in tech and contributing to a diverse, inclusive, equitable
           tech environment</li>
-        <li>have less than the amount specified in your location and situation on the <a
+        <!-- In light of the tuition change, do we need to change the wording for this bullet?  -->
+        <!-- <li>have less than the amount specified in your location and situation on the <a
```

Comment:
Just remove this as we mention elsewhere how this makes people eligible to apply for scholarships and stipends, right?

Reply:
### ✅ Addressed in [1e45bab](https://github.com/Techtonica/techtonica.org/pull/336/commits/1e45bab21cda780bd785d464f8c358bf46a9f5c1).